src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -224,11 +231,11 @@ private MessageConsumer newInitialStream(
         {
             final long newTraceId = supplyTrace.getAsLong();
             final long acceptReplyId = supplyReplyId.applyAsLong(acceptInitialId);
-
+            final long challengeDelta = resolveChallengeDelta(verified, expiresAtMillis);
```

Comment:
Please rename all references to `challenge delta` to `challenge response timeout`, in all fields, methods, variables, etc.